troubadix/argparser.py

```diff
@@ -230,6 +235,17 @@ def parse_args(
         help="Don't print the statistic",
     )
 
+    parser.add_argument(
+        "--plugins-config-file",
+        type=file_type,
+        default="troubadix_config.toml",
```

Comment:
```suggestion
        default="troubadix.toml",
```
What do you think about this? It aligns a little more with other configs like `poetry.toml` or `pyproject.toml`, `Cargo.toml`

Reply:
sounds good